packages/guideline-themes-settings/package.json

```
@@ -0,0 +1,58 @@
+{
+    "name": "@frontify/guideline-themes-settings",
+    "type": "module",
+    "version": "0.0.0",
+    "description": "Provides types and helpers for the guideline themes development",
+    "author": "Frontify Developers <developers@frontify.com>",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Frontify/brand-sdk",
+        "directory": "packages/guideline-themes-settings"
+    },
+    "sideEffects": false,
+    "publishConfig": {
+        "access": "public"
+    },
+    "exports": {
+        ".": {
+            "types": "./dist/index.d.ts",
+            "require": "./dist/index.umd.js",
+            "import": "./dist/index.es.js"
+        }
+    },
+    "main": "./dist/index.es.js",
+    "types": "./dist/index.d.ts",
+    "files": [
+        "dist"
+    ],
+    "scripts": {
+        "build": "vite build",
+        "lint": "eslint .",
+        "lint:fix": "eslint --fix .",
+        "prettier": "prettier --check .",
+        "prettier:fix": "prettier --write .",
+        "test": "vitest run --silent",
+        "test:coverage": "vitest --coverage",
+        "test:watch": "vitest",
+        "test:ui": "vitest --ui",
+        "typecheck": "tsc --noEmit"
+    },
+    "dependencies": {
+        "@frontify/sidebar-settings": "^0.9.20"
+    },
+    "devDependencies": {
+        "@frontify/app-bridge-theme": "0.0.0-alpha.17",
```

If [this comment](https://github.com/Frontify/brand-sdk/pull/1087/files#r1735928135) isn't applicable, then we need to do this and also move it to `dependencies`:
```suggestion
        "@frontify/app-bridge-theme": "workspace:^",
```